cli/tracker.php

```diff
@@ -17,6 +17,11 @@
 ini_set('display_errors', 1);
 
 define('JPATH_ROOT', realpath(__DIR__ . '/..'));
+define('JPATH_CONFIGURATION', JPATH_ROOT . '/etc');
```

Comment:
👍 on that.  I only noticed because I ran the migration on my local to make sure I copied everything from the issue text correctly.